frappe_telegram/client.py

```diff
@@ -80,3 +89,48 @@ def get_bot(telegram_bot) -> Bot:
     return ExtBot(
         token=telegram_bot.get_password("api_token")
     )
+
+
+def send_message_from_template(template: str, context: dict = {}, lang: str = None,
+                               parse_mode=None, user=None, telegram_user=None, from_bot=None):
+    """
+    Use a Telegram Message Template to send a message
+
+    template: `str`
+        Name of a Telegram Message Template
+    context: `dict`
+        dict of key:values to reslove the tags in the template
+    lang: `str`
+        Optionally can be set if an alternative template language is needed
+    """
+
+    dt = "Telegram Message Template"
+
+    filters = [
+        ["name", "=", template],
+    ]
+    if lang:
+        filters.append(["Telegram Message Template Translation", "language", "=", lang])
+
+    templates = frappe.get_all(
+        dt,
+        filters=filters
+    )
+
+    if not templates:
+        if lang:
+            frappe.throw(_("No template with name '{0}' and language '{1}' exists.").format(
```

Comment:
@Abadulrehman can't we use `default_template` when no template exists for a particular language ?

Reply:
Yeah, that seems like the better option.